bin/merge_describer

```diff
@@ -1,23 +1,28 @@
 #!/bin/sh
+# A script to parse git log output and generate a list of PRs that have been merged.
+# Useful for PR descriptions when merging in changes from nucore-open.
+# This is also a dependency of the release_describer script.
+#
 # Usage:
 # Compare your current branch to master:
 # bin/merge_describer
 #
-# Compare your current branch to master:
-# bin/merge_describer master
+# Compare your current branch to a tag:
+# bin/merge_describer v2024-06-18
 target_branch=${1:-master}
 git log $target_branch.. | egrep '\(#\d+\)$' | while read LINE; do
-  pr=$(echo $LINE | sed -E 's/^.+\(#//' | sed -E 's/\)$//')
-  desc=$(echo "$LINE" | sed -E 's/ *\(#[0-9]+\)$//' | sed -E 's/[*]//')
-  echo "$pr\t$desc"
-done | sort -n |
-
-while read LINE; do
```

Comment:
I'm not sure why we were iterating through this twice before, so I decided to simplify